src/System.Private.ServiceModel/tools/scripts/WcfTestServerSetup.cmd

```diff
@@ -0,0 +1,191 @@
+@echo off
+setlocal
+
+set Id=%1
```

Comment:
Nit: I know setlocal is used, but I'd prefer if these environment variables were prefixed with underscore so it is clear when reading later areas of the script that you are referring only to local environment variables.